mocha-multi.js

```diff
@@ -74,17 +79,16 @@ MochaMulti.prototype.done = function (failures, fn) {
 };
 
 const msgs = {
-  no_definitions: "reporter definitions should be set in \
-the `multi` shell variable\n\
-eg. `multi='dot=- xunit=file.xml' mocha`",
-  invalid_definition: "'%s' is an invalid definition\n\
-expected <reporter>=<destination>",
+  no_definitions: 'reporter definitions should be set in ' +
```

Comment:
should probably use a template string here?

Reply:
no, template strings preserve source newlines.

Reply:
aha, of course.